### PR TITLE
Add the NHS to the used and loved by section

### PR DESCRIPTION
**Pull request by tynandebold**
Signed-off-by: Tynan DeBold <thdebold@gmail.com>

## Description

@datajoely found out that the NHS [uses Kedro](https://nhsx.github.io/skunkworks/synthetic-data-pipeline)! This adds them to the **Used and loved by** section of the website.

## QA notes

image

Question from me: is it simply **NHS** or **The NHS**?

## Checklist

- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Added tests to cover my changes, where applicable

### CODE REVIEW

**Comment by tynandebold** [comment]
> Can we not make this hyperlink to the article?

We could. @GabrielComymQB would probably need to provide some style for hovering over the company name though? If not, these can all easily be links with the same style the have at this moment in time.

**Comment by GabrielComymQB** [comment]
> 

Hey, I guess for now we can do a subtle hover playing with the alpha value, but keeping the same yellow colour. 
We can make it FFBC00 as the main, and when you hover on "NHS" for example, it goes FFBC00 , alpha .85
A bit of a freestyle solution for now.

**Comment by datajoely** [comment]
If helpful, these are links for the other projects:

- [Leapfrog](https://www.lftechnology.com/blog/ai-pipeline-kedro/)
- [Telkomsel](https://medium.com/life-at-telkomsel/how-we-build-a-production-grade-data-pipeline-7004e56c8c98)
- [NASA](https://github.com/nasa/ML-airport-configuration)
- [Belfius](https://www.linkedin.com/posts/vangansen_mlops-machinelearning-kedro-activity-6772379995953238016-JUmo/)
- [Sber](https://www.linkedin.com/posts/seleznev-artem_welcome-to-kedros-documentation-kedro-activity-6767523561109385216-woTt/?ref=morioh.com&utm_source=morioh.com)
- [Augment Partners](https://www.linkedin.com/posts/augment-partners_kedro-cheat-sheet-by-augment-activity-6858927624631283712-Ivqk/?ref=morioh.com&utm_source=morioh.com)
- [McKinsey & Company](https://www.mckinsey.com/alumni/news-and-insights/global-news/firm-news/kedro-from-proprietary-to-open-source)
- [GetInData](https://getindata.com/blog/running-machine-learning-pipelines-kedro-kubeflow-airflow)
- [Indicium](https://medium.com/indiciumtech/how-to-build-models-as-products-using-mlops-part-2-machine-learning-pipelines-with-kedro-10337c48de92)
- [GMO](https://recruit.gmo.jp/engineer/jisedai/engineer/jisedai/engineer/jisedai/engineer/jisedai/engineer/jisedai/blog/kedro_and_mlflow_tracking/)
- [QuantumBlack](https://medium.com/quantumblack/introducing-kedro-the-open-source-library-for-production-ready-machine-learning-code-d1c6d26ce2cf)
- [XP](https://youtu.be/wgnGOVNkXqU?t=2210)
- [AI Singapore](https://makerspace.aisingapore.org/2020/08/leveraging-kedro-in-100e/)
- [Helvetas](https://www.linkedin.com/posts/lionel-trebuchon_mlflow-kedro-ml-ugcPost-6747074322164154368-umKw)
- [Leapfrog](https://www.lftechnology.com/blog/ai-pipeline-kedro/)
- [Naranja](https://medium.com/quantumblack/odelsa-uses-kedro-for-social-good-3c7e7fc20306)
- [Beamery](https://medium.com/hacking-talent/production-code-for-data-science-and-our-experience-with-kedro-60bb69934d1f)

File: components/companies-using-kedro/companies-using-kedro-content.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  {
+    name: 'GetInData',
+    link: 'https://getindata.com/blog/running-machine-learning-pipelines-kedro-kubeflow-airflow',
+  },
+  {
+    name: 'Indicium',
+    link: 'https://medium.com/indiciumtech/how-to-build-models-as-products-using-mlops-part-2-machine-learning-pipelines-with-kedro-10337c48de92',
+  },
+  {
+    name: 'NHS',
```
Comment (yetudada) [inline components/companies-using-kedro/companies-using-kedro-content.tsx:36]:
```suggestion
    name: 'NHS AI Lab',
```

**Review by yetudada — APPROVED** [review]
I'll just be specific about the part of the NHS that uses Kedro but otherwise all happy with the changes 🥳 I like the linking thing.

**Pull request merged**